src/jit/decomposelongs.cpp

```diff
@@ -140,6 +136,7 @@ GenTree* DecomposeLongs::DecomposeNode(LIR::Use& use)
     }
 #endif // DEBUG
 
+    GenTree* nextNode = nullptr;
```

Comment:
I thought you changed the semantics to not allow nullptr return. This will be a problem for GT_PHI/GT_PHI_ARG.